spyder/plugins/editor/lsp/client.py

```diff
@@ -124,7 +124,7 @@ def start(self):
                 ' '.join(self.server_args)))
             creation_flags = 0
             if WINDOWS:
-                creation_flags = subprocess.CREATE_NEW_PROCESS_GROUP
+                creation_flags = 0x08000000 # CREATE_NO_WINDOW
```

Comment:
Why not append to the previous flag instead of replacing it entirely? What does `subprocess.CREATE_NEW_PROCESS_GROUP` does?

Reply:
Totally right, `CREATE_NEW_PROCESS_GROUP` makes possible to use os.kill() in the subprocess

Reply:
This is what the docs say: 

>Windows: The signal.CTRL_C_EVENT and signal.CTRL_BREAK_EVENT signals are special signals which can only be sent to console processes which share a common console window, e.g., some subprocesses. Any other value for sig will cause the process to be unconditionally killed by the TerminateProcess API, and the exit code will be set to sig. The Windows version of kill() additionally takes process handles to be killed.

Reply:
Ok, thanks!